ovos_dinkum_listener/voice_loop/hotwords.py

```diff
@@ -204,11 +204,12 @@ def found(self) -> Optional[str]:
         if self.state == HotwordState.LISTEN:
             engines = self.listen_words
             if not engines:
-                LOG.debug("No listen_words loaded")
+                raise RuntimeWarning(
+                    f"Waiting for listen_words but none are available!")
         elif self.state == HotwordState.WAKEUP:
             engines = self.wakeup_words
             if not engines:
-                LOG.debug("No wakeup_words loaded")
+                LOG.warning("Waiting for wakeup_words but none are available!")
```

Comment:
wont this also log in a loop? should only warn once... on init should be enough

Reply:
Moved all logging to `load_hotword_engines`. I left the runtime error to runtime to handle cases where `HotwordContainer` might be used without a wake word being required.